portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,33 +40,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     
-  // stores both the name and the comment
-  private List<Pair<String, String>> commentData; 
   private int maxComments;
 
   @Override
   public void init() {
-    commentData = new ArrayList<Pair<String, String>>();
     maxComments = 10; // arbitrary default value
```

Comment:
instead of this, you can just have a static class member for default value, so the value won't have to be initialized every time:
private static final int MAX_COMMENTS = 10

Reply:
will fix it now!